src/views/Layout.jsx

```diff
@@ -15,6 +15,23 @@ import {
 import logo from '../images/PredictaCartLogo.png';
 
 export function Layout() {
+	const location = useLocation();
+	let tabIndex;
+
+	switch (location.pathname) {
```

Comment:
`location.pathname`: The location object returned by `useLocation` has various properties. One of these is `pathname`, which is a string representing the current URL's path. 

For example, if the current URL is `https://example.com/list`, then `location.pathname` would be `/list`.